splinterd/api/static/openapi.yml

```diff
@@ -327,6 +327,145 @@ paths:
         500:
           description: Internal service error
 
+  /user/notifications:
+    get:
+      description: Retrieve a list of all notifications
+      responses:
+        200:
+          description: Notifications have been retrieved
+          content:
+            application/json:
+              schema:
+                type: array
+                $ref: '#/components/schemas/Notification'
+        403:
+          description: User not authorized to access this endpoint
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    post:
```

Comment:
Should a notification be posted to a particular user? Like POST `/user/{user}/notification` instead of the general notification endpoint. Or maybe both options should exist.

Reply:
I don't think so. A notification can be sent to a list of recipients